src/server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const fileUpload = require('express-fileupload');
+const fs = require('fs');
+const index = require('./index');
```

Comment:
We can rename the `index.js` to `cli.js`.

We could split `index.js: run()`:

1. file handling (read/write to JSON/CSV)
2. string handling + logic (transforming logic)

E.g.

```
function run(csvFilePath, timesheetJsonPath, userType) {
  // fetch csv data into memory
  const transformedCsvData = transform(csvData, timesheetJSON, userType)
  // write to file
}

But I think this might be out of the scope of this PR.